gems/sidekiq-pro/OSVDB-126331.yml

```diff
@@ -0,0 +1,11 @@
+---
+gem: sidekiq-pro
+osvdb: 126331
+url: http://osvdb.org/show/osvdb/126331
+title: Sidekiq Pro Gem for Ruby CSRF in Job Filtering
+date: 2015-07-17
+description: |
+  Sidekiq::Web job filtering lacks CSRF protection. This issue
+  is related to OSVDB-125675.
```

Comment:
Do we want to use the (undocumented) `related` field for this? Basically, Sidekiq Pro depends on Sidekiq, so the fix in Sidekiq (OSVDB-125675) is needed for this fix in Sidekiq Pro to work correctly.